### ISSUE TITLE
Learning rate

### ISSUE THREAD

**Issue by gissemari** [phreeza/tensorflow-vrnn#5]
Hi phreeza,
The learning rate is initialized as` self.lr = tf.Variable(0.0, trainable=False)` and not taken from the parameters. It should be initialized with the parameter and then modified with the decay right?
